pepper-apis/src/main/java/org/broadinstitute/ddp/housekeeping/PubSubMessageBuilder.java

```diff
@@ -152,6 +153,7 @@ public PubsubMessage createMessage(String ddpMessageId,
                                 .findProfileByUserId(gov.getProxyUserId()).orElse(null);
                         if (profile != null) {
                             queuedNotificationDto.addTemplateSubstitutions(
+                                    new NotificationTemplateSubstitutionDto(DDP_PROXY_GUID, gov.getProxyUserGuid()),
```

Comment:
There's a bit of an issue using `proxy_firstName` in the emails. The name is collected only after consent is submitted. So for emails like "welcome" and "consent reminder" we don't have the name. Instead, it seems way better to use `proxy_guid` instead. Adding this seems straightforward, so I made the change.

Reply:
> Instead, it seems way better to use proxy_guid instead

In email it shown as a real GUID value?

Reply:
We're not displaying the GUID in the email. Rather we're just using it to do conditional logic in the email template.